doc/Native APIs.md

```diff
@@ -0,0 +1,417 @@
+Native API Documentation
+========================
+
+This documents the APIs that can be used to implement native wrappers. There are
+three types of APIs:
+
+- `student_view_data`: Exposes block settings and content as JSON data. Can be
+  retrieved via the edX Course Blocks API. Does not include user-specific data
+  (i.e. items derived from `Scope.user_state` fields).
```

Comment:
Thanks! I will update according to your suggestions. The example in the Course API docs actually uses both,`requested_fields=student_view_data` and `student_view_data=<block-type>`. I *think* I had to use both when playing with the API on edx/master some time ago, but maybe I just never tried without `requested_fields`.